api/pkg/notification/notification.go

```
@@ -0,0 +1,56 @@
+	req, err := http.NewRequest("POST", url, strings.NewReader(string(jsonData)))
+	if err != nil {
+		return fmt.Errorf("failed to create notification request: %v", err)
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := n.client.Do(req)
+	if err != nil {
+		return fmt.Errorf("failed to send notification: %v", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (1.62.2)

Error return value of `resp.Body.Close` is not checked

(errcheck)